core/include/scipp/core/variable.h

```diff
@@ -52,7 +52,9 @@ template <class... Known> class VariableConceptHandle_impl;
   double, float, int64_t, int32_t, bool, Eigen::Vector3d,                      \
       sparse_container<double>, sparse_container<float>,                       \
       sparse_container<int64_t>, sparse_container<int32_t>,                    \
-      span<const double>, span<double>, span<const float>, span<float>
+      sparse_container<bool>, span<const double>, span<double>,                \
```

Comment:
It should be minor, because adding more types here just increases the number of variant alternatives. It does *not* change the number of cases instantiated by `transform` (unless we add the new types to the list).